package.json

```diff
@@ -108,12 +108,12 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^29.7.0",
+    "jest": "^28.1.3",
```

Comment:
Let's use the latest jest as it is a dev dependency it won't affect the users work with older node versions.